.github/workflows/notebooks.yml

```diff
@@ -16,15 +22,32 @@ jobs:
 
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.10
+
+    - name: Set up Python
       uses: actions/setup-python@v3
       with:
         python-version: ${{ matrix.python-version }}
+
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip --quiet install --upgrade .[complete]
-        python -m pip list
+        python -m pip install --upgrade .[test]
+
+    - name: List installed Python packages
+      run: python -m pip list
+
+    - name: REMOVE test a failing test by self
+      shell: bash
+      run: |
+        export PYTHONWARNINGS='default'
+        pytest --override-ini filterwarnings= tests/test_notebooks.py -k test_xml_importexport
```

Comment:
This is just here to understand what is happening in Issue #1840 at the single test level. This should be removed after that.